tests/test_2024_03_18.py

```diff
@@ -115,8 +115,8 @@ def test_vote(helpers, accounts, vote_ids_from_env, bypass_events_decoding, stra
     )
 
     # Check Oracle Config state before voting
-    finalization_max_negative_rebase_epoch_shift = convert.to_uint(oracle_daemon_config.get("FINALIZATION_MAX_NEGATIVE_REBASE_EPOCH_SHIFT"))
-    assert finalization_max_negative_rebase_epoch_shift == FINALIZATION_MAX_NEGATIVE_REBASE_EPOCH_SHIFT_OLD_VALUE
+    finalization_max_negative_rebase_epoch_shift = oracle_daemon_config.get("FINALIZATION_MAX_NEGATIVE_REBASE_EPOCH_SHIFT")
+    assert finalization_max_negative_rebase_epoch_shift.hex() == FINALIZATION_MAX_NEGATIVE_REBASE_EPOCH_SHIFT_OLD_VALUE
```

Comment:
can we avoid convert to hex here? why do we need it?

Reply:
Can't avoid hex here. It's needed to convert value from int to the format contract getter returns.